contents/articles/2017-01_render-visual-field/index.md

```diff
@@ -0,0 +1,22 @@
+--- 
+title: Adapt Rendering to the User's Current Visual Field
+date: Mon Jan 09 2017 17:13:14 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Daniel Pohl et al., propose to use the user's visual field of the eye gaze for render optimizations and increase performance..." 
```

Comment:
Daniel Pohl et al., use Pupil Labs' Oculus DK2 add-on to propose a rendering technique for HMDs that is optimized  based on the user's current visual field...